operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -1099,6 +1141,24 @@ func (r *Cluster) PandaproxyAPIInternal() *PandaproxyAPI {
 	return nil
 }
 
+// AllPandaproxyAPIExternalListeners returns all the external pandaproxy listeners
+func (r *Cluster) AllPandaproxyAPIExternalListeners() []PandaproxyAPI {
```

Comment:
DITTO as with `ExternalListener` and `AllKafkaAPIExternalListeners`. The `PandaproxyAPIExternal` could be renamed to `FirstExternalPandaproxyAPIListener` and this function could be named `ExternalPandaproxyAPIListener`.